Reduce the test tolerance a little due to some failing regression tests
(see https://github.com/ludwig-cf/ludwig/issues/367)

diff -rupN ludwig-ludwig-0.23.1.orig/tests/awk-fp-diff.sh ludwig-ludwig-0.23.1/tests/awk-fp-diff.sh
--- ludwig-ludwig-0.23.1.orig/tests/awk-fp-diff.sh	2025-05-07 15:17:41.000000000 +0200
+++ ludwig-ludwig-0.23.1/tests/awk-fp-diff.sh	2025-07-08 12:26:37.245077520 +0200
@@ -34,7 +34,7 @@ BEGIN {
   # files2[]   lines of file 2 (1..nlines2 with file2[0] the filename)
   # lcslen[,]  lowest common subsequence array for diff algorithm 
 
-  TOLERANCE = 1.0e-12
+  TOLERANCE = 1.0e-11
   nlines1 = 0
   nlines2 = 0
   file1[0] = ARGV[1]
